### PR TITLE
Remove faulty validation for VPA

### PR DESCRIPTION
**Pull request by bskiba**
The CRD creation doesn't work at the moment. Validation fails for enums and lists. This should fix e2e tests for now. 
Will add validation to admission webhook in a followup pr.

### CODE REVIEW

**Comment by bskiba** [comment]
@schylek

File: vertical-pod-autoscaler/deploy/vpa-crd.yaml [hunk 1]
```diff
@@ -26,31 +26,11 @@ spec:
...
-                  enum:
-                    - "Off"
-                    - "Initial"
-                    - "Recreate"
-                    - "Auto"
             resourcePolicy:
               properties:
                 containerPolicies:
                   type: array
                   items:
```
Comment (schylek) [inline vertical-pod-autoscaler/deploy/vpa-crd.yaml]: After removing lines below items will be empty. Should be removed too?

Comment (bskiba) [reply vertical-pod-autoscaler/deploy/vpa-crd.yaml]: Done

**Pull request merged**